internal/api/users/list_test.go

```diff
@@ -0,0 +1,135 @@
+package users
+
+import (
+	"reflect"
+	"slices"
+	"testing"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/model"
+)
+
+func TestList(test *testing.T) {
+	expectedUsers := []core.ServerUserInfo{
```

Comment:
Note that you are testing the API endpoint `users/list`.
You are NOT testing the functionality of pulling users from the database (leave that to `internal/db` to test).

Simplify this test to assess the API endpoint instead of the DB.